samila/functions.py

```diff
@@ -261,7 +291,9 @@ def filter_size(size):
 def plot_params_filter(
         g,
         color=None,
+        c=None,
```

Comment:
Why?
I think that the presence of `c` and `color` at the same time confuses the user!